### PR TITLE
trivial: Use the correct branch when building a flatpak bundle

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: contrib/ci/flatpak.sh [hunk 1]
```diff
@@ -2,13 +2,33 @@
 set -e
 set -x
 
+# get the correct branch to use
+if [ -z "$FWUPD_BRANCH" ]; then
+    if [ -n $TRAVIS_BRANCH ]; then
```
[the remark this answers is not in the document]
Comment (hughsie) [reply contrib/ci/flatpak.sh]: `FWUPD_BRANCH` is the thing we're setting, `TRAVIS_BRANCH` is the thing that might exist.

File: contrib/ci/flatpak.sh [hunk 2]
```diff
@@ -2,13 +2,33 @@
...
+
 # install the runtimes
 flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
 flatpak install flathub runtime/org.gnome.Sdk/x86_64/3.28
 flatpak install flathub runtime/org.gnome.Platform/x86_64/3.28
 
 # build the bundle
-flatpak-builder --repo=repo --force-clean --disable-rofiles-fuse build-dir contrib/org.freedesktop.fwupd.json
+envsubst < contrib/org.freedesktop.fwupd.json > /tmp/org.freedesktop.fwupd.json
+flatpak-builder --repo=repo --force-clean --disable-rofiles-fuse build-dir /tmp/org.freedesktop.fwupd.json
```
[the remark this answers is not in the document]
Comment (hughsie) [reply contrib/ci/flatpak.sh]: Sure, can do.

File: contrib/ci/flatpak.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (superm1) [reply contrib/ci/flatpak.sh]: Oh right, reading. Sorry about that.

File: contrib/ci/flatpak.sh [hunk 3]
```diff
@@ -2,13 +2,37 @@
...
+
 # install the runtimes
 flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
 flatpak install flathub runtime/org.gnome.Sdk/x86_64/3.28
 flatpak install flathub runtime/org.gnome.Platform/x86_64/3.28
 
-# build the bundle
-flatpak-builder --repo=repo --force-clean --disable-rofiles-fuse build-dir contrib/org.freedesktop.fwupd.json
+# copy in the branch
+rm -rf tmp
```
Comment (superm1) [inline contrib/ci/flatpak.sh]: Can you make this build/ instead?  It's already gitignored and used by all the other build scripts.

Comment (hughsie) [reply contrib/ci/flatpak.sh]: Done.

**Pull request merged**